client/src/__tests__/ListLatestEntries.test.jsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import ListLatestEntries from '../components/ListLatestEntries';
+
+beforeAll(() => {
+  // Mock the global fetch function with a custom implementation
+  globalThis.fetch = async () => {
+    return {
+      json: async () => [
+        {
+          entry_id: 1,
+          entry_type: 'Rose',
+          entry_content: 'Some rose content',
+        },
+        {
+          entry_id: 2,
+          entry_type: 'Bud',
+          entry_content: 'Some bud content',
+        },
+      ],
+    };
+  };
+});
+
+test('list entries on page', async () => {
+    render(
+    <Router>
+        <ListLatestEntries />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!